sample-schemas/demo_schema.erl

```diff
@@ -0,0 +1,66 @@
+-module(demo_schema).
+
+-include_lib("typerefl/include/types.hrl").
+
+-type birthdays() :: [#{m := 1..12, d := 1..31}].
+
+-reflect_type([birthdays/0]).
+
+-export([namespaces/0, fields/1]).
+
+namespaces() -> ["foo", "a.b", "x.y"].
```

Comment:
we should perhaps only export root level, or re-usable namespaces here ?
if `x.y` and `j.k` are only internally used, there is no need to expose them.